packages/@webex/internal-plugin-dss/src/dss.ts

```diff
@@ -163,17 +166,23 @@ const DSS = WebexPlugin.extend({
    * @returns {Array} result.foundArray an array of the lookups of the found entities (if foundPath provided)
    * @returns {Array} result.notFoundArray an array of the lookups of the not found entities (if notFoundPath provided)
    */
-  _request(options) {
-    const {resource, params, dataPath, foundPath, notFoundPath} = options;
+  _request(options: RequestOptions): Promise<RequestResult> {
+    const {resource, params, dataPath, foundPath, notFoundPath, timeout} = options;
 
     const requestId = uuid.v4();
     const eventName = this._getResultEventName(requestId);
     const result = {};
-    let expectedSeqNums;
-    let notFoundArray;
+    let expectedSeqNums: string[];
+    let notFoundArray: unknown[];
+
+    return new Promise((resolve, reject) => {
+      const timer = new Timer(() => {
+        this.stopListening(this, eventName);
+        reject(new DssTimeoutError({requestId, timeout, resource, params}));
```

Comment:
the timeout in the error should also default to the config one, just tried it and got "webex-web-client Module:DSS Error:The KMS did not respond within undefined ms."